pkg/fleet/bootstraper/bootstraper.go

```diff
@@ -0,0 +1,120 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package bootstraper provides the installer bootstraper component.
+package bootstraper
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/DataDog/datadog-agent/pkg/fleet/internal/exec"
+	"github.com/DataDog/datadog-agent/pkg/fleet/internal/oci"
+)
+
+const (
+	installerPackage        = "datadog-installer"
+	installerPackageVersion = "latest"
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 Suggestion:** ‏I don't see a way to override this but this will be needed for tests. Do you mind adding an option & a command line option for this?

Reply:
added